testkit/src/simple_supervisor/tests.rs

```diff
@@ -425,7 +425,7 @@ fn service_config_discard_fake_supervisor() {
 fn test_configuration_and_rollbacks() {
     let mut testkit = TestKitBuilder::auditor()
         .with_validators(1)
-        .with_service(SimpleSupervisor)
+        .with_simple_supervisor()
```

Comment:
I think that `with_rust_service(SimpleSupervisor)` is enough.

Reply:
Well, I added it because `SimpleSupervisor` is not used outside TestKit and this method simplifies its usage a little. But in fact, it's not such important and can be deleted.